### PR TITLE
Add Sends API in bitwarden-uniffi

### PR DESCRIPTION
**Pull request by dani-garcia**
## Type of change
```
- [ ] Bug fix
- [x] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective
Add missing Sends API to the bitwarden-uniffi crate.

Also fixed copy-pasted comment in the password history function

### CODE REVIEW

**Review by Hinton — COMMENTED** [review]
Can you re-generate the doc file?

**Comment by dani-garcia** [comment]
Done, I also updated the prettierignore so that `npm run prettier` works on the file.

Not sure why but when I ran the doc generation the ExportFormat definition got deleted from the doc, so I restored it by hand.

File: .prettierignore [hunk 1]
```diff
@@ -1,5 +1,8 @@
 target
-languages
+languages/*
+!/languages/kotlin
+languages/kotlin/*
```
Comment (Hinton) [inline .prettierignore:4]: Is this needed?

Comment (dani-garcia) [reply .prettierignore:4]: It's a bit awkward but that seems to be how the ignore file works yeah:
https://github.com/prettier/prettier/issues/3328#issuecomment-350732773

We could always remove it and add an extra step on the Readme to format the doc file after it's generated.

**Review by Hinton — APPROVED** [review]

**Pull request merged**